packages/cli/src/utils.ts

```diff
@@ -323,3 +351,7 @@ export function isSubdir(parent: string, dir: string): boolean {
   const relative = path.relative(parent, dir);
   return !!relative && !/^..($|\/)/.test(relative) && !path.isAbsolute(relative);
 }
+
+export function isURL(str: string): boolean {
+  return /^(https?:)\/\//m.test(str);
```

Comment:
What if it's an http url?

Reply:
It will pass, but there is another check that already exist and can be reused.